packages/theme/src/components/form.ts

```diff
@@ -2,7 +2,7 @@ import { mode } from "@chakra-ui/theme-tools"
 
 type Dict = Record<string, any>
 
-const parts = ["requiredIndicator", "helperText"]
+const parts = ["formControl", "requiredIndicator", "helperText"]
```

Comment:
```suggestion
const parts = ["container", "requiredIndicator", "helperText"]
```

We _mostly_ have a pattern of calling the root element a `container`. Could you update this to follow that pattern?

Reply:
👍🏽